tree/BaseTree.gd

```diff
@@ -0,0 +1,12 @@
+extends StaticBody2D
+class_name BaseTree
```

Comment:
Any reason this would be inheritance based rather than using composition? 

Basically is there ever giong to be a tree that doesnt produce sunlight?